docs/source/notebooks/wps_decompose_flow_vectors_demo.ipynb

```diff
@@ -20,12 +20,30 @@
     "from wps_tools.utils import copy_http_content\n",
```

Comment:
<p>The URL still targets the prod version. Is there something I need to do before running the demo?</p><pre class="ql-syntax" spellcheck="false">Using thunderbird on https://docker-dev03.pcic.uvic.ca/twitcher/ows/proxy/thunderbird/wps
</pre>
<br/>

 _Reply via <a href='https://app.reviewnb.com/pacificclimate/thunderbird/pull/127/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='2'/>

Reply:
Are you using `make test-notebooks-custom`?